### ISSUE TITLE
removeUnusedNamespaces fails to recognise used namespaces in certain cases

### ISSUE THREAD

**Issue by codeworrior** [kristian/minify-xml#4]
The [regular expression](https://github.com/kristian/minify-xml/blob/master/index.js#L94) which is used to detect the used tag namespaces fails when there is a sequence of opening tags without namespace, e.g. for an XML like

```xml
<App><Page><customHeader><f:ShellBar title="{i18n>TITLE}" homeIcon="./img/logo_ui5.png"/>
```

The string `<App><Page><customHeader><f` is extracts as a used namespace. But only `f` should be extracted.

I think, the character '>' needs to be added to the character set of the RegExp.

Also see https://github.com/SAP/ui5-builder/issues/538

**Comment by kristian** [comment]
Good point. Let me do the fix.

**Comment by codeworrior** [comment]
BTW: while looking into this, I noticed that we refer to minify-xml@2.1.2, but there's no tag or release yet with that version in this repository.

**Comment by kristian** [comment]
> BTW: while looking into this, I noticed that we refer to minify-xml@2.1.2, but there's no tag or release yet with that version in this repository.

this I fixed right away. let me check the issue with the namespaces now

**Comment by kristian** [comment]
Fixed with 2.1.3
